### ISSUE TITLE
Schema

### ISSUE THREAD

**Issue by wipfli** [loeli-style/loeli-style#9]
Should we use an existing schema? Shortbread and protomaps come to mind. Shortbread is CC0. What is protomaps, @bdon?

https://github.com/protomaps/protomaps-themes#readme

**Comment by bdon** [comment]
@wipfli CC0 - I have some plans to evolve this in the next few months, and the intention is to offer a general layer of interoperability between tiles and styles, with easy extension points for additional layers. I think we should coordinate!

**Comment by wipfli** [comment]
Sounds good, @bdon. I think we can already start putting some basic stuff together. We can always adjust the layers and style...

**Comment by wipfli** [comment]
Thanks @bdon for open-sourcing the protomaps tiles at https://github.com/protomaps/basemaps/tree/main/tiles

I did a first run of planetiler and the tiles look great. Here is Monaco:

![image](https://user-images.githubusercontent.com/53421382/222908396-1e6ebf09-7961-431c-a42f-7ec591f2bbf7.png)

**Comment by wipfli** [comment]
@maxammann I think we should just build on top of the protomaps tiles. They are CC0. What do you think?

**Comment by maxammann** [comment]
Absolutely! I think we can try to stay compatible but not force ourselves too much.

It is definitely a good starting point.
